13/umbraco-cms/tutorials/add-google-authentication.md

```diff
@@ -42,68 +41,65 @@ The first thing to do is set up a Google API. To do this, you need to go to [htt
 
 1. Click the project dropdown and select **New Project**.
 
-<figure><img src="images/Project_dropdown_list.png" alt=""><figcaption></figcaption></figure>
+    ![Project dropdown list](images/Project_dropdown_list_v13.png)
 
 2. Enter a **Project name**, **Organization**, and **Location**.
-3. **Create** the project.
-
-<figure><img src="images/Project_Details.png" alt=""><figcaption></figcaption></figure>
+3. Click **Create**.
 
 ### Enable the Google+ API
 
 1. Open the newly created project from the project dropdown.
-2. Select **Enable APIs and Services**.
+2. Click **Enable APIs and Services**.
 
-<figure><img src="images/Enable_Apis.png" alt=""><figcaption></figcaption></figure>
+    ![Enable APIs](images/Enable_Apis_v13.png)
 
-2. Use the search field to find the **Google+ API**.
-3. **Enable** the product to enable the API.
+3. Type **Google+ API** in the **Search** field.
+4. Click **Enable**.
```

Comment:
just checked and first you need to click on it and only after you get the choice to enable it
```suggestion
4. Select it and then **Enable** it.
```